### PR TITLE
Adding XStatic-Bootstrap

### PR DESCRIPTION
**Pull request by ReimarBauer**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by ReimarBauer** [comment]
@conda-forge/staged-recipes please review this one!

File: recipes/xstatic-bootstrap/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1800e6bb5fc687604d8a893eee8c7882d800a6f3d6721799016f99a47d1dac0f
+
+build:
+  number: 0
```
Comment (beckermr) [inline recipes/xstatic-bootstrap/meta.yaml:13]: This recipe can be noarch: python

File: recipes/xstatic-bootstrap/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
```
Comment (beckermr) [inline recipes/xstatic-bootstrap/meta.yaml:20]: It is very likely that you do not need setuptools here as it is pip dependency.

File: recipes/xstatic-bootstrap/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - xstatic.pkg.bootstrap
+
+about:
+  home: https://getbootstrap.com/
```
Comment (beckermr) [inline recipes/xstatic-bootstrap/meta.yaml:29]: This homepage should be for the python package, not bootstrap.

File: recipes/xstatic-bootstrap/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Twitter Bootstrap packaged for setuptools'
+
+  description: |
+    Quickly design and customize responsive mobile-first sites with Bootstrap, the world’s most popular front-end open source toolkit, featuring Sass variables and mixins, responsive grid system, extensive prebuilt components, and powerful JavaScript plugins.
+  doc_url: https://getbootstrap.com
```
Comment (chrisburr) [inline recipes/xstatic-bootstrap/meta.yaml:37]:
```suggestion
  doc_url: https://xstatic.readthedocs.io/
```

File: recipes/xstatic-bootstrap/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+  dev_url: https://github.com/twbs/bootstrap
```
Comment (chrisburr) [inline recipes/xstatic-bootstrap/meta.yaml:38]: I think this should probably be:
```suggestion
  dev_url: https://github.com/xstatic-py/xstatic-bootstrap
```

Comment (beckermr) [inline recipes/xstatic-bootstrap/meta.yaml:38]: Same for these thinks.

File: recipes/xstatic-bootstrap/meta.yaml (hunk 3 above)
Comment (chrisburr) [inline recipes/xstatic-bootstrap/meta.yaml:29]:
```suggestion
  home: https://xstatic.readthedocs.io/
```

File: recipes/xstatic-bootstrap/meta.yaml (hunk 5 above)
Comment (ReimarBauer) [reply recipes/xstatic-bootstrap/meta.yaml:38]: I see, looking for solving this.

Comment (ReimarBauer) [reply recipes/xstatic-bootstrap/meta.yaml:38]: ah ok, that's a start, I asked Thomas if he has access to the recent one.

**Comment by ReimarBauer** [comment]
Hello
@beckermr @chrisburr 

now better state? And can it be merged please.

cheers
Reimar

**Comment by ReimarBauer** [comment]
@conda-forge/core please merge

File: recipes/xstatic-bootstrap/meta.yaml (hunk 1 above)
Comment (ocefpaf) [reply recipes/xstatic-bootstrap/meta.yaml:13]: Quick question @ReimarBauer. Did you create this recipe with [grayskull](https://pypi.org/project/grayskull/)? It should've done the `noarch` suggestion for you there.

**Pull request merged**